test/models/pawn_test.rb

```diff
@@ -1,4 +1,102 @@
 require 'test_helper'
 
 class PawnTest < ActiveSupport::TestCase
+  def setup
+    @game = Game.new
+    @game.populate_board!
+    @first_white_pawn = Pawn.create(game: @game, color: 'white', x_position: 3, y_position: 6)
```

Comment:
Ok. For some weird reason, I though I need to use the method populate_board! 
After all, I didn't need to use that method.
Thanks!